src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+
+    private static final int MIN_COUNT_OF_PERSON = 2;
+    private final List<Point> points;
```

Comment:
다른 클래스처럼 클래스 변수와 멤버 변수는 개행으로 구분하면 가독성에 도움이 될 것 같아요
```suggestion
    private static final int MIN_COUNT_OF_PERSON = 2;
    
    private final List<Point> points;
```